components/Toc.tsx

```diff
@@ -46,7 +47,8 @@ const TocContainer = ({ activeHeading, activeSubHeading, CurrentDocsSlug }) => {
                     </div>
                 </>
             ) : null}
-
+            <hr />
```

Comment:
for server-side API pages, the "run in postman" button is not vertically centered

<img width="393" alt="Screenshot 2022-12-14 at 12 06 40 AM" src="https://user-images.githubusercontent.com/284612/207384182-e235c8b3-7e47-4d9d-a8b3-820a51b29216.png">